tests_e2e/test_suites/images.yml

```diff
@@ -52,10 +61,14 @@ images:
 # TODO: Add CentOS 6.10 and Debian 8
 #
 #   centos_610: "OpenLogic CentOS 6.10 latest"
-   centos_79: "OpenLogic CentOS 7_9 latest"
 #   debian_8: "credativ Debian 8 latest"
+#
+   alma_9: "almalinux:almalinux:9-gen2:latest"
```

Comment:
can we keep same format across all distros? or is it supposed to be that way?

Reply:
yes, this was a copy-n-paste and forgot to replace the :

i want to use spaces all over because improves readibility. i'll update this one

Reply:
btw (LISA accepts both, CLI requires colon)

Reply:
I see few other places too; can you also update those? Thanks.

Reply:
updated